packages/docker-engine/docker-engine.spec

```diff
@@ -32,6 +32,8 @@ Source1000: clarify.toml
 # Bottlerocket-specific - Privileged containers should receive SELinux labels
 # https://github.com/bottlerocket-os/bottlerocket/issues/1011
 Patch0001: 0001-bottlerocket-privileged-shouldn-t-disable-SELinux.patch
+# Update aws-sdk-go for IMDSv2 support
```

Comment:
```suggestion
Patch0001: 0001-bottlerocket-privileged-shouldn-t-disable-SELinux.patch

# Update aws-sdk-go for IMDSv2 support
```
This isn't Bottlerocket-specific, so we want it to visually appear as a different section.